core/controllers/suggestion_test.py

```diff
@@ -188,6 +188,49 @@ def test_create_suggestion(self):
         self.assertEqual(len(suggestions), 3)
         self.logout()
 
+    def test_edit_suggestion(self):
+
+        suggestion = self.get_json(
+            '%s?author_id=%s' % (
+                feconf.SUGGESTION_LIST_URL_PREFIX,
+                self.author_id_2))['suggestions'][1]
+
+        # Create new suggestion content.
+        new_content_after_edit = state_domain.SubtitledHtml(
+            'content', 'new edit content html').to_dict()
```

Comment:
no

Reply:
can you show where we are handling this case?

Reply:
no, I have not handled this case, but before moving further can you tell me how a user can change content id. AFAIK content id is created in behind the scenes and user have no option in GUI to change content id and I think content id also fixed when some content is created.
If I am missing something please tell me!!

Reply:
Ah, from the backend perspective think you are creating an API, and GUI is just making it easier for users to interact with the API, but the user can directly call API through HTTP request.

Also, I'm asking this because we aren't showing content ids to the reviewer and if they merged the suggestion with wrong content_ids then this will make the entire exploration fail the validation check.

(Note: I'm not sure whether we validate the exploration with the suggestions before allowing the reviewer to accept the suggestion.)

Reply:
@DubeySandeep this should be checked by the accept suggestion function, not by edit suggestion function.
In edit suggestion, we were just updating the existing suggestion.

Reply:
So can we update the existing content id in a suggestion through editing?